server/src/resolvers/answer.js

```diff
@@ -11,13 +15,24 @@ module.exports = {
     createAnswerAndSources: async (_, { content, sources, nodeId }, ctx, info) => {
       sources = JSON.parse(sources)
 
+      let language = ''
+      let translation = { language: language, text: '' }
+      if (process.env.CLOUD_TRANSLATION_API_KEY) {
+        language = await detectLanguage(content)
+        const { targetLanguage, translatedText } = await getTranslatedText(content, language)
+        translation = keyValuePairsToTranslations(
+          translationToKeyValuePairs(targetLanguage, translatedText)
+        )
```

Comment:
Pourquoi pas plus simplement ceci :
```suggestion
        translation = await getTranslatedText(content, language)
```